zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/InsertTraceIdByAnnotation.java

```diff
@@ -1,58 +0,0 @@
-/*
- * Copyright 2015-2019 The OpenZipkin Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
-package zipkin2.storage.cassandra.v1;
-
-import com.datastax.driver.core.BoundStatement;
-import com.datastax.driver.core.querybuilder.Insert;
-import com.datastax.driver.core.querybuilder.QueryBuilder;
-import java.util.Set;
-import java.util.concurrent.ThreadLocalRandom;
-import zipkin2.Span;
-
-import static zipkin2.storage.cassandra.v1.CassandraUtil.annotationKeys;
-import static zipkin2.storage.cassandra.v1.CassandraUtil.toByteBuffer;
-
-// QueryRequest.annotations/binaryAnnotations
-final class InsertTraceIdByAnnotation implements Indexer.IndexSupport {
-  private static final ThreadLocalRandom RAND = ThreadLocalRandom.current();
```

Comment:
specifically.. here is where we oddly saved off a "thread local random".. maybe the point was to save a random (regardless, it should be field not constant)

Reply:
@michaelsembwever if you happen to remember why we did this, let me know. I've left it in for now.